clang/lib/CodeGen/TargetBuiltins/AMDGPU.cpp

```diff
@@ -564,6 +564,11 @@ Value *CodeGenFunction::EmitAMDGPUBuiltinExpr(unsigned BuiltinID,
     llvm::Function *F = CGM.getIntrinsic(IID, {LoadTy});
     return Builder.CreateCall(F, {Addr});
   }
+  case AMDGPU::BI__builtin_amdgcn_load_to_lds: {
+    // Should this have asan instrumentation?
```

Comment:
Repeating comment from the PR that duplicated this one, yes but that's a backend problem?

Reply:
(Bumping this comment)